origin/test/extended/util/pods.go

```diff
@@ -75,10 +76,22 @@ func RemovePodsWithPrefixes(oc *CLI, prefixes ...string) error {
 // The security context of this pod complies to the "restricted" profile.
 // If necessary this can be overriden in tweaks.
 func CreateExecPodOrFail(client kubernetes.Interface, ns, name string, tweak ...func(*v1.Pod)) *v1.Pod {
+	return createExecPodOrFail(client, ns, name, image.ShellImage(), tweak...)
+}
+
+// CreateAgnhostPodOrFail creates a pod used as a vessel for kubectl exec commands.
+// Pod name is uniquely generated.
+// The security context of this pod complies to the "restricted" profile.
+// If necessary this can be overriden in tweaks.
+func CreateAgnhostPodOrFail(client kubernetes.Interface, ns, name string, tweak ...func(*v1.Pod)) *v1.Pod {
+	return createExecPodOrFail(client, ns, name, podframework.GetTestImage(imageutils.Agnhost), tweak...)
```

Comment:
Why do we need image from K8s? What's wrong with the `image.ShellImage()`? (preferably added to PR desc or commit msg)

Reply:
That one lacks the tools these tests are using, thats why I opted for agnhost, which is also upstreamed.